docs/en/reference/api/containers/optioncontainer.md

```diff
@@ -14,7 +14,7 @@ A container that can display multiple labeled tabs of content.
 
 ///
 
-/// tab | Linux
+/// tab | GTK
```

Comment:
I'd lean towards "Linux (GTK)" as the label here, as we don't support GTK on non-Linux platforms (similarly for "Linux (Qt)")